cdap-ui/app/directives/widget-container/widget-schema-editor/widget-schema-editor.js

```diff
@@ -392,6 +392,13 @@ angular.module(PKG.name + '.commons')
           EventPipe.cancelEvent('dataset.selected');
           URL.revokeObjectURL($scope.url);
         });
+
+        $scope.loadNextSetOfRows = _.debounce(function() {
+          if ($scope.properties.length - $scope.limitedToView > 10) {
```

Comment:
It seems like the logic in here and in `cdap-ui/app/features/hydratorplusplus/controllers/create/partials/nodeconfig-ctrl.js` do not handle the case where there are a number of properties that is not like 15, 25, 35, 45, etc...

For instance, if there are 16 properties, initially it will load 15. Then, it'll come to this `if` condition and check `16 - 15 > 10`, and that will be false, so it wont increase the `limitedToView` limit.

Essentially, it fails to load the last few properties, if the remainder of properties is less than 10.
Let me know if I'm misunderstanding that.